### DISCUSSION TITLE
Update Lexical to v0.13

### DISCUSSION THREAD

**Discussion by vlad-zhukov** [mdx-editor/editor#377] [General]
First of all, thank for the great library!

Do you have any plans to update Lexical to [v0.13](https://github.com/facebook/lexical/releases/tag/v0.13.0)? It would be nice because it fixes a few bugs (for example, [this one](https://github.com/facebook/lexical/issues/5369)).

And followup questions:
- Do you think it's relatively trivial or do you see any potential problems?
- Is it a major change for the MDXEditor?

**Comment by petyosi** [comment]
This goes without saying. MDXEditor is an actively developed component, and Lexcal is its bread and butter. The last lexical upgrade was actually contributed :). If you want to, you can give it a try.
